services/httpd/default.nix

```diff
@@ -28,9 +26,9 @@ let
     documentRoot = "${common.webtreeDir}/redbrick/htdocs";
   in {
     inherit adminAddr documentRoot;
-    hostName = tld;
-    listen = [{ port = 443; }];
```

Comment:
same question as above

Reply:
Yeah onlySSL covers this, as mentioned above